### PR TITLE
consider adding position_name, height_ft and name

### PR DESCRIPTION
**Pull request by CodyStringham**

### CODE REVIEW

**Comment by CodyStringham** [comment]
Not sure why the age one came in again, let me know if its causing problems and i'll re-clone and do a new PR.

File: lib/sportradar/api/soccer/player.rb [hunk 1]
```diff
@@ -38,17 +42,23 @@ def initialize(data)
...
       def full_name
-        full = [full_first_name, full_last_name].join(' ')
-        full == " " ? name : full
+        full = [@full_first_name, @full_last_name].join(' ')
+        full == " " ? name : @full
       end
 
       def position_name
         positions = {"G" => "Goalie", "D" => "Defender", "M" => "Midfielder", "F" => "Forward"}
-        positions[position] if position
```
Comment (beneggett) [inline lib/sportradar/api/soccer/player.rb:51]: You should keep this part here still for the match_summary calls, which include the position during the game.

so maybe do an 

```
if position
    positions[position]
elsif teams
  ... your code .... 
end 
```

**Comment by CodyStringham** [comment]
updated.

**Comment by beneggett** [comment]
Thanks

**Pull request merged**